modules/renter.go

```diff
@@ -157,6 +152,10 @@ type Renter interface {
 	// Contracts returns the contracts formed by the renter.
 	Contracts() []RenterContract
 
+	// CurrentPeriod returns the height at which the current allowance period
+	// began.
+	CurrentPeriod() types.BlockHeight
```

Comment:
A name like `CurrentPeriod()` makes me expect a range, but I guess this is okay.